src/api/voting.ts

```diff
@@ -124,6 +124,22 @@ export type ProcessKeys = {
     revealKeys?: { idx: number, key: string }[]
 }
 
+export type ProcessEnvelope = {
```

Comment:
```suggestion
export type EnvelopeMeta = {
```

Reply:
Do we need to export these types?
They pollute the exported namespace and I doubt that someone needs it